Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml

```
@@ -790,6 +790,7 @@
   description: A kitchen and restaurant equipment vendor.
   components:
   - type: VendingMachine
+    customWindowTitle: Plasteel Chef Dinnerware
```

```suggestion
    customWindowTitle: Plasteel Chef's Dinnerware
```
also this needs to be localized